src/auto-compounder/interfaces/Slipstream/ISwapRouter.sol

```diff
@@ -0,0 +1,30 @@
+/**
+ * Created by Pragma Labs
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity 0.8.22;
+
+interface ISwapRouter {
+    struct ExactInputSingleParams {
```

Comment:
General styling, we mix (in all interfaces so far) structs declared inside and outside of the interface.
Would pick one from now on (prefer declared outside and imported directly into the main contracts).